libra_toolbox/neutron_detection/activation_foils/foils.py

```diff
@@ -0,0 +1,198 @@
+        """
+        self.mass = mass
+        self.thickness = thickness
+        self.name = name
+        self.half_life = None
+        self.decay_constant = None
+        self.reaction = None
+        self.branching_ratio = None
+        self.photon_energies = None
+        self.cross_section = None  # in barns (1 barn = 1e-24 cm^2)
+        self.density = None  # in g/cm^3
+        self.atomic_mass = None
+        self.abundance = None
+        self.mass_attenuation_coefficient = None  # in cm^2/g
+    
+    def get_properties(self):
+        """Returns the properties of the foil."""
+        return {
+            "Name": self.name,
+            "Mass (g)": self.mass,
+            "Thickness (cm)": self.thickness,
+            "Decay Constant (1/s)": self.decay_constant,
+            "Reaction": self.reaction,
+            "Branching Ratio": self.branching_ratio,
+            "Photon Energies": self.photon_energies,
+            "Cross Section (barns)": self.cross_section,
+            "Density (g/cm³)": self.density,
+            "Atomic Mass": self.atomic_mass,
+            "Abundance": self.abundance,
+            "Mass Attenuation Coefficient (cm²/g)": self.mass_attenuation_coefficient
+        }
+    
+    def get_decay_constant(self):
+        self.decay_constant = np.log(2) / self.half_life.to(ureg.s)
+
+    def get_atoms(self):
+        self.atoms = (self.abundance 
+                      * (self.mass 
+                         * (6.022141e23 * ureg.particle / ureg.mol) 
+                         / self.atomic_mass))
+
+class Niobium(Foil):
+    def __init__(self, mass: float, thickness: float, name=''):
+        super().__init__(mass, thickness, name=name)
+
+        self.half_life = np.array([10.15]) * ureg.day
+        self.get_decay_constant()
+
+        self.reaction = ["Nb93(n,2n)Nb92m"]
+        self.branching_ratio = np.array([0.9915])
+        self.photon_energies = np.array([934.44]) * ureg.keV
+
+        # Cross section from EAF-2010 at 14 MeV
+        self.cross_section = np.array([0.4729]) * ureg.barn
+
+        self.density = 8.582 * ureg.g / ureg.cm**3 
+        self.atomic_mass = 92.90637 * ureg.g/ureg.mol
+        # Natural abundance of reactant in analyzed foil reaction
+        self.abundance = np.array([1.00])
+        self.get_atoms()
+
+        # Should update mass attenuation coefficient to be photon energy-dependent
+        self.mass_attenuation_coefficient = np.array([0.06120]) * ureg.cm**2 / ureg.g   # at 1 MeV
+
+    def get_100mL_params(self):
+        # Adjust parameters to replicate previous 100 mL results
+
+        # half life used in BABY 100 mL analysis
+        self.half_life = np.array([10.25]) * ureg.day
+        self.get_decay_constant()
+
+        # branching ratio used in BABY 100 mL analysis
+        self.branching_ratio = np.array([0.999])
+
+        # cross section used in BABY 100 mL analysis
+        self.cross_section = np.array([0.46]) * ureg.barn
```

Comment:
Apart from branching ratio, I don't see why these parameters should change outside of the 100 mL experiment?